Components/common/SpeechToText/SpeechToText.js

```diff
@@ -0,0 +1,182 @@
+/**
+ * @flow
+ */
+
+import React, { Component } from 'react';
+import { Text, View } from 'react-native';
+import { Audio } from 'expo-av';
+import * as FileSystem from 'expo-file-system';
+import * as Font from 'expo-font';
+import * as Permissions from 'expo-permissions';
+import { PropTypes } from 'prop-types';
+import { postBlob } from '../../../Utils/postBlob';
+import Button from '../Button/Button';
+import styles from './styles';
+
+export class SpeechToText extends Component {
```

Comment:
@kalex19 

I think this is a great idea to separate SpeechToText into its own component. It will probably make testing easier for the other components.

Reply:
Yeah, exactly. It makes it easier for us to integrate into tasks, etc.